_content/monographs/brief-intro-to-buddha-dhamma_payutto.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "A Brief Introduction to Buddha-Dhamma"
+authors:
+  - payutto
```

Comment:
The build failure says `Failed to find author pages for: payutto` I assume this error would go away with a rebase.